### PR TITLE
EHN: also raise B006 for list/dict/set comprehensions

### PR DESCRIPTION
**Pull request by ichard26**
Fixes GH-184.

### CODE REVIEW

**Comment by ichard26** [comment]
oh bloody hell, there's AST differences on Python 3.7 and below >.<

File: bugbear.py [hunk 1]
```diff
@@ -653,6 +655,7 @@ def visit(self, node):
     )
 )
 B006.mutable_literals = (ast.Dict, ast.List, ast.Set)
+B006.mutable_comprehensions = (ast.ListComp, ast.DictComp)
```
Comment (isidentical) [inline bugbear.py:658]: I think set comprehensions (`{x for x in y}`) is also suitable in this context
```suggestion
B006.mutable_comprehensions = (ast.ListComp, ast.DictComp, ast.SetComp)
```

Comment (ichard26) [reply bugbear.py:658]: And I knew I was forgetting something, thanks for noticing I'll fix it soon once I get back to my computer at home.

**Review by cooperlees — APPROVED** [review]
LGTM - Thanks Mr @ichard26 :D

**Pull request merged**